src/Raven.Server/Documents/Revisions/RevisionsStorage.cs

```diff
@@ -1470,14 +1471,24 @@ public async Task<IOperationResult> RevertRevisions(DateTime before, TimeSpan wi
             // send initial progress
             parameters.OnProgress?.Invoke(result);
 
+            HashSet<string> collectionsHashSet = null;
+            if (collections != null)
+            {
+                collectionsHashSet = new HashSet<string>();
```

Comment:
Using `StringComparer.OrdinalIgnoreCase`, instead.

Also, if you hare using `HashSet` here, why accept a `List` above?

Reply:
Done